urlparse4/cgurl.pyx

```diff
@@ -8,10 +8,19 @@ import six
 from six.moves.urllib.parse import urlsplit as stdlib_urlsplit
 from six.moves.urllib.parse import urljoin as stdlib_urljoin
 from six.moves.urllib.parse import urlunsplit as stdlib_urlunsplit
+from six.moves.urllib.parse import urlparse as stdlib_urlparse
+from six.moves.urllib.parse import urlunparse as stdlib_urlunparse
 
 cimport cython
 
 
+uses_params = [scheme.encode('utf-8') for scheme in ['', 'ftp', 'hdl',
```

Comment:
can be written as ``[b'', b'ftp', ...``, without the need to call encoding, I think it will be much more clear